probe/docker/controls.go

```diff
@@ -99,7 +99,7 @@ func (r *registry) execContainer(containerID string, req xfer.Request) xfer.Resp
 		AttachStdout: true,
 		AttachStderr: true,
 		Tty:          true,
-		Cmd:          []string{"/bin/sh"},
+		Cmd:          []string{"/usr/bin/env", "TERM=xterm", "/bin/sh", "-l"},
```

Comment:
Why do we want a login shell? `-l` doesn't seem to be POSIX compliant.

Also, busybox doesn't ship `/usr/bin/env` but `/bin/env`